automation/docker/run-tests.sh

```diff
@@ -4,12 +4,19 @@
 NUM_NODES="${1:-4}"  # default to 4
 ENV="${2:-dev}"  # default to dev
 export ENV=$ENV
+HUB_COMPOSE=hub-compose.yml
 
 #if test runs against a remote FIAB on a GCE node, put IP in param 3
 #if test runs against a local FIAB on a Dokcer, put "local" in param 3
 #else leave blank (test runs against a non FIAB host, such as real dev)
 DOCKERHOST="127.0.0.1"
 DOCKERHOST=${3:-$DOCKERHOST}
```

Comment:
Since there are 5 positional parameters, is it possible for $3 to be blank now?

Reply:
no, talked with jackie and agreed that it would be nice to use flags to only specify the params you want but i didn't want to tackle that for this PR. also, didn't want to rearrange params because that causes jenkins to break and everyone has to rebase their branches to get them to work.